ios/xcode/urlSession/src/js_Foundation/js_NSURLSessionConfiguration_P.m

```diff
@@ -1920,7 +1920,16 @@ JSValueRef backgroundSessionConfigurationForNSURLSessionConfigurationConstructor
     {
     	bool identifier$0$free = false;
     NSString * identifier$0 = HyperloopJSValueRefToNSString(ctx,arguments[0],exception,&identifier$0$free);
-    	NSURLSessionConfiguration * result$ = [NSURLSessionConfiguration backgroundSessionConfiguration:identifier$0];
+        NSURLSessionConfiguration * result$;
+        //check if >iOS8
+    if ([UIView instancesRespondToSelector:@selector(layoutMarginsDidChange)])
+    {
+        result$ = [NSURLSessionConfiguration backgroundSessionConfigurationWithIdentifier:identifier$0];
+    }
+    else 
+    {
+    	result$ = [NSURLSessionConfiguration backgroundSessionConfiguration:identifier$0];
+    }
```

Comment:
@cheekiatng - This fix needs to be done in the [app.hjs](https://github.com/appcelerator-modules/ti.urlsession/blob/master/ios/js/app.hjs) file. This is just a generated class.